include/EffectView.h

```diff
@@ -90,6 +96,9 @@ public slots:
 	Knob * m_gate;
 	QMdiSubWindow * m_subWindow;
 	EffectControlDialog * m_controlView;
+	
+	bool m_dragging;
+	QGraphicsOpacityEffect * m_opacityEffect;
```

Comment:
I remember we discussed this somewhere, but I don't think we ever put it in the style guide itself. It might be part of the clang format check.
```suggestion
	QGraphicsOpacityEffect* m_opacityEffect;
```